db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "The internal database ID."
+    change_column_comment :miq_tasks, :identifier, "The resource this task is related to in '#{klass}:#{id}' format."
+    change_column_comment :miq_tasks, :message, "Message that describes the reason for the current status."
+    change_column_comment :miq_tasks, :miq_server_id, "ID of the ManageIQ server that the task belongs to."
```

Comment:
it is id of server where task was executed

Reply:
Ok, updated.